### ISSUE TITLE
"Expected mp3, got something else. Aborting mp3 time calculation. GOT: " "C:\\Users\\''my user''\\AppData\\Local\\osu!\\Songs\\circles.ogg"

### ISSUE THREAD

**Issue by Tassynary** [l3lackShark/gosumemory#66]
![sdsddssdds](https://user-images.githubusercontent.com/81843126/113467431-579e0880-9419-11eb-9d43-e47be5f78cf9.png)
sometimes when I open my game it crashes and the overlay doesn’t work at all, I did all the correct procedures, but it just doesn’t work

**Comment by RoARene317** [comment]
Are your song in osu! Directory use weird character (Like non english character)?

**Comment by Tassynary** [comment]
> Are your song in osu! Directory use weird character (Like non english character)?

no bro

**Comment by UpsetDavid** [comment]
I also have this issue happen frequently. Unsure what causes it - but it seems to happen when osu! crashes (the crash happens regardless of if gosumemory is running or not). 

![](https://cdn.discordapp.com/attachments/517502416900456450/828149734371622922/unknown.png)

Due to my game crashing regardless of gosumemory's running state, I doubt this is a problem with gosumemory. However, when it crashes, I can consistently see this error.

**Comment by Tassynary** [comment]
my osu is crashing only when i run gosumemory with active overlay, so I believe the problem is with the gosumemory or something missing in osu! folders

**Comment by l3lackShark** [comment]
This particular error has nothing to do with the crash.  We just don't support .ogg calculation and circles.ogg does not physically exist on the system (probably baked in some dll or resource file)

**Comment by l3lackShark** [comment]
> my osu is crashing only when i run gosumemory with active overlay, so I believe the problem is with the gosumemory or something missing in osu! folders

Are you running with in-game oerlay? Does it crash instantly?

**Comment by Tassynary** [comment]
yes, the gosumemory works normally without the active overlay, but when I activate it the game crashes or the overlay just doesn't appear

**Comment by Tassynary** [comment]
> > my osu is crashing only when i run gosumemory with active overlay, so I believe the problem is with the gosumemory or something missing in osu! folders
> 
> Are you running with in-game oerlay? Does it crash instantly?

basically if i enable the overlay the game crashes, but if i open the gosu memory without activating the overlay it works normally

**Comment by Tassynary** [comment]
> > my osu is crashing only when i run gosumemory with active overlay, so I believe the problem is with the gosumemory or something missing in osu! folders
> 
> Are you running with in-game oerlay? Does it crash instantly?

I already tried re-installing gosumemory, and nothing worked, I tried a few other things and it didn't work, I just don't want to uninstall osu because I have a lot of maps, replays and skins, but if that is necessary I might re-install osu (and sorry if you can't understand something, my english is not very good)

**Comment by l3lackShark** [comment]
> basically if i enable the overlay the game crashes, but if i open the gosumemory without activating the overlay it works normally

Do you play with compatibility mode enabled?

**Comment by Tassynary** [comment]
> > basically if i enable the overlay the game crashes, but if i open the gosumemory without activating the overlay it works normally
> 
> Do you play with compatibility mode enabled?

no

**Comment by l3lackShark** [comment]
Windows 10? What graphics card? And once again, to reiterate. It crashes instantly or you can see the overlay for some time?

**Comment by Tassynary** [comment]
yes windows 10, my graphics card is a gtx 1660 Ti and it crashes instantly and i can't see the overlay

**Comment by l3lackShark** [comment]
Got any antivirus running in the background?

**Comment by Tassynary** [comment]
> Got any antivirus running in the background?

no, i only use windows defender

**Comment by l3lackShark** [comment]
Please attach a screenshot of your `config.ini`

**Comment by Tassynary** [comment]
![image](https://user-images.githubusercontent.com/81843126/113572851-60483780-95ef-11eb-936a-b7607965dc9a.png)


> Please attach a screenshot of your `config.ini`

**Comment by Mrcubix** [comment]
I can say that this issue doesn't only happen if the game crash, the error will eventually occur during long sessions and gosumemory will start spamming "no files or such directory" and won't work as intended anymore after that, default config.

**Comment by l3lackShark** [comment]
This particular error has nothing to do with the crash. We just don't support ogg time calc yet.

**Comment by HollisMeynell** [comment]
I ran into the same problem, and osu must crash, the graphics card is AMD

**Comment by HollisMeynell** [comment]
![image](https://user-images.githubusercontent.com/56613764/123749247-cb03c900-d8e7-11eb-8762-d49815389102.png)

**Comment by HollisMeynell** [comment]
> I ran into the same problem, and osu must crash, the graphics card is AMD
> Start osu first and then gosumemory ,osu will immediately crash
> Start gosumemory first and then osu, osu crashes after the "welcome to osu" sound ends
> Stopping the Game Overlay game will be normal
I find a debugfile,This might help you
[0629/134348.448:ERROR:main_delegate.cc(714)] Could not load locale pak for en-US
[0629/134348.448:ERROR:main_delegate.cc(721)] Could not load cef.pak
[0629/134348.448:ERROR:main_delegate.cc(738)] Could not load cef_100_percent.pak
[0629/134348.449:ERROR:main_delegate.cc(747)] Could not load cef_200_percent.pak
[0629/134348.449:ERROR:main_delegate.cc(757)] Could not load cef_extensions.pak
[0629/134348.466:ERROR:content_client.cc(276)] No data resource available for id 101
[0629/134427.281:ERROR:main_delegate.cc(714)] Could not load locale pak for en-US
[0629/134427.281:ERROR:main_delegate.cc(721)] Could not load cef.pak
[0629/134427.281:ERROR:main_delegate.cc(738)] Could not load cef_100_percent.pak
[0629/134427.281:ERROR:main_delegate.cc(747)] Could not load cef_200_percent.pak
[0629/134427.281:ERROR:main_delegate.cc(757)] Could not load cef_extensions.pak
[0629/134427.295:ERROR:content_client.cc(276)] No data resource available for id 101
[0629/134744.636:ERROR:main_delegate.cc(714)] Could not load locale pak for en-US
[0629/134744.636:ERROR:main_delegate.cc(721)] Could not load cef.pak
[0629/134744.636:ERROR:main_delegate.cc(738)] Could not load cef_100_percent.pak
[0629/134744.636:ERROR:main_delegate.cc(747)] Could not load cef_200_percent.pak
[0629/134744.636:ERROR:main_delegate.cc(757)] Could not load cef_extensions.pak
[0629/134744.648:ERROR:content_client.cc(276)] No data resource available for id 101
[0629/135857.539:ERROR:main_delegate.cc(714)] Could not load locale pak for en-US
[0629/135857.539:ERROR:main_delegate.cc(721)] Could not load cef.pak
[0629/135857.539:ERROR:main_delegate.cc(738)] Could not load cef_100_percent.pak
[0629/135857.539:ERROR:main_delegate.cc(747)] Could not load cef_200_percent.pak
[0629/135857.539:ERROR:main_delegate.cc(757)] Could not load cef_extensions.pak
[0629/135857.552:ERROR:content_client.cc(276)] No data resource available for id 101
[0629/140148.831:ERROR:main_delegate.cc(714)] Could not load locale pak for en-US
[0629/140148.832:ERROR:main_delegate.cc(721)] Could not load cef.pak
[0629/140148.832:ERROR:main_delegate.cc(738)] Could not load cef_100_percent.pak
[0629/140148.832:ERROR:main_delegate.cc(747)] Could not load cef_200_percent.pak
[0629/140148.832:ERROR:main_delegate.cc(757)] Could not load cef_extensions.pak
[0629/140148.844:ERROR:content_client.cc(276)] No data resource available for id 101
[0629/141127.347:ERROR:main_delegate.cc(714)] Could not load locale pak for en-US
[0629/141127.347:ERROR:main_delegate.cc(721)] Could not load cef.pak
[0629/141127.347:ERROR:main_delegate.cc(738)] Could not load cef_100_percent.pak
[0629/141127.347:ERROR:main_delegate.cc(747)] Could not load cef_200_percent.pak
[0629/141127.347:ERROR:main_delegate.cc(757)] Could not load cef_extensions.pak
[0629/141127.360:ERROR:content_client.cc(276)] No data resource available for id 101
[0629/141154.990:ERROR:main_delegate.cc(714)] Could not load locale pak for en-US
[0629/141154.990:ERROR:main_delegate.cc(721)] Could not load cef.pak
[0629/141154.990:ERROR:main_delegate.cc(738)] Could not load cef_100_percent.pak
[0629/141154.990:ERROR:main_delegate.cc(747)] Could not load cef_200_percent.pak
[0629/141154.990:ERROR:main_delegate.cc(757)] Could not load cef_extensions.pak
[0629/141155.003:ERROR:content_client.cc(276)] No data resource available for id 101
[0629/142857.350:ERROR:main_delegate.cc(714)] Could not load locale pak for en-US
[0629/142857.350:ERROR:main_delegate.cc(721)] Could not load cef.pak
[0629/142857.351:ERROR:main_delegate.cc(738)] Could not load cef_100_percent.pak
[0629/142857.351:ERROR:main_delegate.cc(747)] Could not load cef_200_percent.pak
[0629/142857.351:ERROR:main_delegate.cc(757)] Could not load cef_extensions.pak
[0629/142857.368:ERROR:content_client.cc(276)] No data resource available for id 101
[0629/143037.958:ERROR:main_delegate.cc(714)] Could not load locale pak for en-US
[0629/143037.958:ERROR:main_delegate.cc(721)] Could not load cef.pak
[0629/143037.958:ERROR:main_delegate.cc(738)] Could not load cef_100_percent.pak
[0629/143037.958:ERROR:main_delegate.cc(747)] Could not load cef_200_percent.pak
[0629/143037.958:ERROR:main_delegate.cc(757)] Could not load cef_extensions.pak
[0629/143037.971:ERROR:content_client.cc(276)] No data resource available for id 101
[0629/143911.292:ERROR:main_delegate.cc(714)] Could not load locale pak for en-US
[0629/143911.292:ERROR:main_delegate.cc(721)] Could not load cef.pak
[0629/143911.292:ERROR:main_delegate.cc(738)] Could not load cef_100_percent.pak
[0629/143911.292:ERROR:main_delegate.cc(747)] Could not load cef_200_percent.pak
[0629/143911.292:ERROR:main_delegate.cc(757)] Could not load cef_extensions.pak
[0629/143911.310:ERROR:content_client.cc(276)] No data resource available for id 101
[0629/143957.027:ERROR:main_delegate.cc(714)] Could not load locale pak for en-US
[0629/143957.028:ERROR:main_delegate.cc(721)] Could not load cef.pak
[0629/143957.028:ERROR:main_delegate.cc(738)] Could not load cef_100_percent.pak
[0629/143957.028:ERROR:main_delegate.cc(747)] Could not load cef_200_percent.pak
[0629/143957.028:ERROR:main_delegate.cc(757)] Could not load cef_extensions.pak
[0629/143957.043:ERROR:content_client.cc(276)] No data resource available for id 101
[0629/150217.256:ERROR:main_delegate.cc(714)] Could not load locale pak for en-US
[0629/150217.256:ERROR:main_delegate.cc(721)] Could not load cef.pak
[0629/150217.256:ERROR:main_delegate.cc(738)] Could not load cef_100_percent.pak
[0629/150217.257:ERROR:main_delegate.cc(747)] Could not load cef_200_percent.pak
[0629/150217.257:ERROR:main_delegate.cc(757)] Could not load cef_extensions.pak
[0629/150217.270:ERROR:content_client.cc(276)] No data resource available for id 101

**Comment by Gamingman463** [comment]
Is there any solution to this problem yet? Facing the same problem.
